code/part2_exercises/exercises/functors/functor_exercises.js

```diff
@@ -35,7 +35,7 @@ var ex3 = undefined;
 // Use Maybe to rewrite ex4 without an if statement
 
 var ex4 = function (n) {
-  if (n) { return parseInt(n); }
+  if (n !== null && n !== undefined) { return parseInt(n); }
```

Comment:
Thanks for your suggestion, but I think use a strict way to make up condition will be better, especially for the new souls.